upgrade/test_crossgrade.sh

```diff
@@ -475,8 +482,10 @@ istioInstallOptions
 waitForIngress
 waitForPodsReady "${ISTIO_NAMESPACE}"
 
-# Make a copy of the "from" sidecar injector ConfigMap so we can restore the sidecar independently later.
-echo_and_run kubectl get ConfigMap -n "${ISTIO_NAMESPACE}" istio-sidecar-injector -o yaml > ${TMP_DIR}/sidecar-injector-configmap.yaml
+if [[ "${TEST_SCENARIO}" == "upgrade-downgrade" ]];then
+  # Make a copy of the "from" sidecar injector ConfigMap so we can restore the sidecar independently later.
```

Comment:
nit: `so that`, sorry I might made too many picky comments.

Reply:
Done, thank you for your attention to the details and the quality!